Lessons/Lesson 1/Solutions/lesson1WCY.cs

```diff
@@ -0,0 +1,33 @@
+﻿
+
+using System.Runtime.InteropServices;
+
+Console.WriteLine("Please enter a number: ");
+double x; //declalres an unitialized integer x
+char op;
+double y; 
```

Comment:
you should declare all variables at the top of the file under includes (sing statements)